Dev milestone #2

mistermaxis-zz opened the pull request:
### Changes Implemented:
- Added the three columns required in the second milestone of the project.
- Used bootstrap for style and layout.

mistermaxis-zz committed:
Install and setup linter

mistermaxis-zz committed:
Add images and icons for header

mistermaxis-zz committed:
Add elements for banner part of header

mistermaxis-zz committed:
Add style.css with dummy class

mistermaxis-zz committed:
Add elements for nav part of header.

mistermaxis-zz committed:
Add bootstrap CDN

mistermaxis-zz committed:
Style header and fix linter errors

mistermaxis-zz committed:
Change to header element

mistermaxis-zz committed:
Delete package-lock.json

mistermaxis-zz committed:
Merge branch 'footer' into dev-milestone-1

mistermaxis-zz committed:
Merge branch 'master' into dev-milestone-1

mistermaxis-zz committed:
Deleted hint report html file

mistermaxis-zz committed:
Remove section from header and change date to span

mistermaxis-zz committed:
remove link to stylesheet.css

mistermaxis-zz committed:
Add hint-report to .gitignore

mistermaxis-zz committed:
Add style for footer in style.css

Anas-jaf committed:
added the first culomn and second column but i need to add latest news to it too

IOT-club-mutah committed:
finished the second mileston

mistermaxis-zz committed:
Update index.html

Remove stray "i" tag in line 460 index.html

BrianSammit reviewed (changes requested):
CHANGES REQUIRED 🔴 
------------
Hello, @mistermaxis @luck-i You are doing an amazing job on this milestone. I like the way that you put the content it's really similar to the original one on the desktop size.  👍 💯 
However, there are a few changes to make 🛠️ see them below 👀 👇 
---------------

- [x] Check your commit messages. They must follow the formatting rules. [Here](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) you can read about the formatting rules. [Here](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) you can read about how to changes the commit messages. You don't need to changes the minor errors on the commit messages this time but keep in mind these formating rules for the next commits. 
- [x] Your page is not responsive. Remember that you must follow the same design on mobile and tablet sizes. 
-----------
Make these required changes and send it for another review 😄 
HAPPY CODING!!

![images](https://user-images.githubusercontent.com/64173189/110718644-678b4980-81d9-11eb-9a6e-a7968362da15.jpg)

mistermaxis-zz committed:
Update index.html

Change layout in "main" section so that it responds appropriately to screen size changes.

mistermaxis commented:
### Changes Implemented:

- Checked the commit messages for errors and did research to avoid such errors in the future.
- Changed layout to respond appropriately to screen size changes.

rukundoeric reviewed (approved):
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - [x] The project is clean. 💯
  - [x] The project follows Git flow. 💯

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**
